packages/owl-bot/src/core.ts

```diff
@@ -336,8 +338,11 @@ export async function getAuthenticatedOctokit(
   } else {
     tokenString = auth;
   }
-  const octokit = new Octokit({
+  const OctokitConstructor = shouldRetry ? Octokit.plugin(retry) : Octokit;
+  const octokit = new OctokitConstructor({
     auth: tokenString,
+    doNotRetry: [409],
```

Comment:
Cloud top instances have a lot of hiccups connecting to GitHub, added retry logic so that I can run the population script from my VM.